saas/security/saas_security.xml

```diff
@@ -8,13 +8,14 @@
   <record model="res.groups" id="group_user">
     <field name="name">User</field>
     <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
-    <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    <field name="users" eval="[(4, ref('base.user_demo'))]"/>
```

Comment:
I think it's better to separate user_demo like this: https://github.com/odoo/odoo/blob/12.0/addons/lunch/data/lunch_demo.xml#L6
We have demo directory for it. Here, just as before, use user_root